apis/Google.Cloud.Logging.NLog/Google.Cloud.Logging.NLog/GoogleStackdriverTarget.cs

```diff
@@ -161,7 +161,6 @@ protected override void InitializeTarget()
                                 break;  // Let StringEnumConverter handle formatting
                             return Value.ForNumber(Convert.ToDouble(o));
                         case TypeCode.String:
-                            return Value.ForString((string)o);
                         case TypeCode.Char:
                             return Value.ForString(o.ToString());

```

Comment:
Should this use `Convert.ToString(o))`?

Reply:
Thought object.ToString would be good enough.

Reply:
OK